src/components/VBtn/VBtn.vue

```diff
@@ -126,9 +126,9 @@
       tag === 'button' && (data.attrs.type = this.type)
       this.loading && children.push(this.genLoader())
 
-      if (typeof this.value !== 'undefined') {
-        data.attrs.value = this.value
-      }
+      data.attrs.value = ['string', 'number'].includes(typeof this.value)
+        ? this.value
+        : JSON.stringify(this.value)
```

Comment:
Didn't we talk about this and decided to not stringify the value?

Reply:
Yeah, but `value = false` would result in no value at all, and I figured it'd make sense to have similar behaviour for both radios and button toggles. I did try using `toString` in one of the other commits in this PR, but it was messy and still didn't work properly.